Patch/MinionIdentity_OnSpawn.cs

```diff
@@ -0,0 +1,31 @@
+﻿using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dupery.Patch
+{
+    [HarmonyPatch(typeof(MinionIdentity))]
+    [HarmonyPatch("OnSpawn")]
+    internal class MinionIdentity_OnSpawn
+    {
+        [HarmonyPostfix]
+        static void PostFix(ref MinionIdentity __instance)
+        {
+            SymbolOverrideController component2 = __instance.GetComponent<SymbolOverrideController>();
+            Accessorizer component3 = __instance.gameObject.GetComponent<Accessorizer>();
+
+            Accessory headAccessory = component3.GetAccessory(Db.Get().AccessorySlots.HeadShape);
+            string cheekHash = HashCache.Get().Get(headAccessory.symbol.hash).Replace("headshape", "cheek");

```

Comment:
I'm not yet able to inspect this, I'll DM you about it, but is this line correct? Does the hash string itself contain the substring "headshape"?